### ISSUE TITLE
 High CPU usage

### ISSUE THREAD

**Issue by mobilemindtech** [ChicagoBoss/ChicagoBoss#694]
Hello!

I have two applications using that remain 90% of the time without access, just with running cron jobs. Both applications are all the time occupying over 80% of CPU, often reaching 100% of usage. What can I do to prevent this from happening?

The applications run on a docker, and an EC2 ubuntu.

About the applications: An application sends campaign emails, but it is being used very little ... that is, it almost does not send emails. The other application drives the status of some systems, making https requests periodically.

ChicagoBoss v0.9-beta1
DB MySQL
Session Adapter: mock
Cron plugin: https://github.com/b3rnie/crontab
App runni

Any idea?

Thanks!

**Comment by Shpaky** [comment]
Hi!
Did you solve the problem?
https://github.com/ChicagoBoss/ChicagoBoss/issues/689#issue-516445274

**Comment by mobilemindtech** [comment]
Hi! No, i still have this problem.

**Comment by mobilemindtech** [comment]
I updated my project to OTP 21, and used the following vm.args

```
+sbwt none
+sbwtdcpu none
+sbwtdio none
+sfwi 0
+scl false
-smp
```

It's reduced usage a bit, but it's still high for this application that is idle 90% of the time. The strange thing is that if I run it on my computer the consumption is extremely low. Use 0% most of the time with some peaks of 5% and 7%. But when it is running on an EC2 t3a.xlarge the consumption spikes 100% frequently. Both on my machine and on AWS the application runs in a docker, but on AWS it is inside Swarm. 

I needed to control the processing limit via the swarm configuration, but it's not the best thing to do as the application still consumes unnecessary resources all the time. 

I have no idea what it could be.
